GameData/RealismOverhaul/Engine_Configs/Juno45k_Config.cfg

```diff
@@ -34,7 +34,7 @@
 {
 	%title = Juno IV 45k Engine
 	%manufacturer = JPL
-	%description = When work began on successors to the early Juno launch vehicles, there was an obvious demand for higher performance engines. In order to satisfy this demand, JPL worked on a second stage engine for the Juno IV launch vehicle that would provide 45,000 pounds of thrust in vacuum. The engines were tested, but they were ultimately decided against using them in favor of the X-405 of the Vanguard launch vehicle.
+	%description = Developed by JPL as a higher-performance upgrade for the early Juno engine, this second stage engine was intended for the Juno IV launch vehicle and would provide 45,000 pounds of thrust in vacuum. The engines were tested, but they were ultimately decided against in favor of the X-405.
```

Comment:
Juno was the rocket (and also the engine, the naming scheme is somewhat confusing). This should probably read "Developed by JPL as a high-performance second stage engine for the Juno IV launch vehicle"